ui/app/routes/vault/cluster/settings/auth/enable.js

```diff
@@ -9,11 +9,6 @@ import { inject as service } from '@ember/service';
 export default class VaultClusterSettingsAuthEnableRoute extends Route {
   @service store;
 
-  beforeModel() {
-    // Unload to prevent naming collisions when we mount a new engine
-    this.store.unloadAll('auth-method');
-  }
-
```

Comment:
This was causing the `Expected stable identifier` error in tests which has since been globally fixed but I can't see why this is necessary in the first place. The `path` is used as the unique identifier and there is validation for that so we shouldn't be running into any naming collisions.